shared/chat/inbox/index.native.js

```diff
@@ -85,7 +85,10 @@ class Inbox extends React.PureComponent<Props, State> {
   }
 
   componentWillReceiveProps(nextProps: Props) {
-    if (this.props.rows !== nextProps.rows) {
+    if (
```

Comment:
does anything in this function have to be executed if `nextProps.isActiveRoute` is false? perhaps
```
if (!nextProps.isActiveRoute) {
  return
}
```
at the top.

Otherwise,

```
if (nextProps.isActive && (this.props.rows !== nextProps.rows || !this.props.isActiveRoute)
```

may be clearer

Reply:
this is the entire point of pausableConnect, this should never happen so something else is broken

Reply:
also the rows are equal but failing equality check. i'm fixing this in another branch